kpi-laba-3/solution.py

```diff
@@ -0,0 +1,10 @@
+def main():
+
+    xn_0 = 1
```

Comment:
Лучше использовать осмысленные названия переменных.
```suggestion
    previous = 1
```